network.go

```diff
@@ -200,6 +226,13 @@ func createLink(netHandle *netlink.Handle, name string, expectedLink netlink.Lin
 			LinkAttrs: netlink.LinkAttrs{Name: name},
 			Mode:      netlink.TUNTAP_MODE_TAP,
 		}
+	case (&netlink.Macvtap{}).Type():
+		newLink = &netlink.Macvtap{
+			LinkAttrs:   netlink.LinkAttrs{Name: name},
+			Mode:        netlink.MACVLAN_MODE_BRIDGE,
+			TxQLen:      500,
```

Comment:
Why `500`? Would an admin ever need to change this?

Reply:
@jodh-intel this is the default value that has to be set for packets to flow. The C netlink library sets this. The go netlink library does not and hence needs to be set. No the admin will not need to change this.

Reply:
@jodh-intel changed this to pick the value from the parent. This is the best way to avoid the manifest constant.

Reply:
Great - thanks.